Content.IntegrationTests/Tests/PDA/PDAExtensionsTests.cs

```diff
@@ -1,133 +0,0 @@
-using System.Linq;
-using System.Threading.Tasks;
-using Content.Server.Hands.Components;
-using Content.Server.PDA;
-using Content.Shared.Access.Components;
-using Content.Shared.Containers.ItemSlots;
-using Content.Shared.Inventory;
-using Content.Shared.Item;
-using Content.Shared.PDA;
-using NUnit.Framework;
-using Robust.Server.Player;
-using Robust.Shared.GameObjects;
-
-namespace Content.IntegrationTests.Tests.PDA
-{
-    public sealed class PDAExtensionsTests : ContentIntegrationTest
```

Comment:
Ah yeah I should've made a more detailed PR description, I was pretty tired when I finished ecsing it.

Basically: `PdaExtensions.cs` just provides a `GetHeldId()` function that just obscures some IoC & system resolves. this function is currently only used a single time in `CommunicationsConsoleComponent`.

`IdCardSystem` provides a replacement `TryFindIdCard()`, which does the same thing. So I just removed the extension and the tests, because I CBF fixing all the hand component/system calls.

Reply:
aight